MeruC/PythonProjectG1
# chore/#23 add tailwind

To test this PR:
- Make sure you have NodeJS
- Run `pip install -r requirements.txt`
- Start tailwind on separate terminal - `py manage.py tailwind start`
- For more info on using tailwind: https://clarencedimafelix.notion.site/Using-django-tailwind-caf8a872c5a743939cd2a0c1182eb8f5
